
{'date': 'Mon Jul 27 2020 17:59:41.700 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 26
    main()
  module <module> line 22
    kwarwp_main(Vitollino)
NameError: name 'kwarwp_main' is not defined
'''},
{'date': 'Mon Jul 27 2020 18:09:16.71 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 26
    main()
  module <module> line 22
    kwarwp_main(Vitollino)
  module kwarwp.kwarapp line 34
    Kwarwp(vitollino)
  module kwarwp.kwarapp line 24
    self.cena = self.cria(cenario=cenario) if vitollino else None
  module kwarwp.kwarapp line 28
    cena = self.v.c(self.ABERTURA)
AttributeError: 'Vitollino' object has no attribute 'c'
'''},
{'date': 'Mon Jul 27 2020 18:10:44.370 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    main(None)
  module <module> line 34
    Kwarwp(vitollino)
  module <module> line 23
    self.v = vitollino()
TypeError: 'NoneType' object is not callable
'''},
{'date': 'Mon Jul 27 2020 18:11:09.154 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 26
    main()
  module <module> line 22
    kwarwp_main(Vitollino)
  module kwarwp.kwarapp line 34
    Kwarwp(vitollino)
  module kwarwp.kwarapp line 24
    self.cena = self.cria(cenario=cenario) if vitollino else None
  module kwarwp.kwarapp line 28
    cena = self.v.c(self.ABERTURA)
AttributeError: 'Vitollino' object has no attribute 'c'
'''},
{'date': 'Tue Jul 28 2020 17:05:26.17 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 41
    main(Jogo)
  module <module> line 36
    Kwarwp(vitollino)
  module <module> line 25
    self.cena = self.cria(cenario=cenario) if vitollino else None
  module <module> line 30
    indio = self.v.e(self.INDIO, cena=cena)
AttributeError: 'Jogo' object has no attribute 'e'
'''},
{'date': 'Wed Aug 19 2020 21:05:44.449 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 396
    main(Jogo, STYLE)
  module <module> line 390
    Kwarwp(vitollino, medidas=medidas)
  module <module> line 236
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 270
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 338
    coisa = Piche(imagem, x=0, y=0, cena=cena, taba=self)
  module kwarwp.kwarwpart line 152
    self.posicao = (x//lado,y//lado-1)
TypeError: unsupported operand type(s) for //: 'int' and 'NoneType'
'''},
{'date': 'Wed Aug 19 2020 21:09:00.567 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 396
    main(Jogo, STYLE)
  module <module> line 390
    Kwarwp(vitollino, medidas=medidas)
  module <module> line 236
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 270
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 338
    coisa = Piche(imagem, x=0, y=0, cena=cena, taba=self)
  module kwarwp.kwarwpart line 152
    self.posicao = (x//lado,y//lado-1)
TypeError: unsupported operand type(s) for //: 'int' and 'NoneType'
'''},
{'date': 'Wed Aug 19 2020 21:12:37.628 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 396
    main(Jogo, STYLE)
  module <module> line 390
    Kwarwp(vitollino, medidas=medidas)
  module <module> line 236
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 270
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 338
    coisa = Piche(imagem, x=0, y=0, cena=cena, taba=self)
  module kwarwp.kwarwpart line 153
    self.vazio = Kwarwp.VITOLLINO.a(imagem, w=lado, h=lado, x=0, y=0, cena=cena)
AttributeError: 'NoneType' object has no attribute 'a'
'''},
{'date': 'Wed Aug 19 2020 21:18:46.81 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 396
    main(Jogo, STYLE)
  module <module> line 390
    Kwarwp(vitollino, medidas=medidas)
  module <module> line 236
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 270
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 338
    coisa = Piche(imagem, x=0, y=0, cena=cena, taba=self)
  module kwarwp.kwarwpart line 155
    self.vazio = Kwarwp.VITOLLINO.a(imagem, w=lado, h=lado, x=0, y=0, cena=cena)
AttributeError: 'Jogo' object has no attribute 'a'
'''},
{'date': 'Wed Aug 19 2020 21:20:56.285 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 396
    main(Jogo, STYLE)
  module <module> line 390
    Kwarwp(vitollino, medidas=medidas)
  module <module> line 236
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 270
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 339
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module kwarwp.kwarwpart line 52
    self.posicao = (x//lado,y//lado-1)
TypeError: unsupported operand type(s) for //: 'int' and 'NoneType'
'''},
{'date': 'Fri Aug 21 2020 08:37:14.916 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 396
    main(Jogo, STYLE)
  module <module> line 390
    Kwarwp(vitollino, medidas=medidas).cria()
  module <module> line 270
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 339
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
  module kwarwp.kwarwpart line 54
    self.posicao = (x//lado,y//lado-1)
TypeError: unsupported operand type(s) for //: 'int' and 'NoneType'
'''},
{'date': 'Fri Aug 21 2020 09:27:04.656 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 397
    main(Jogo, STYLE)
  module <module> line 391
    Kwarwp(vitollino, medidas=medidas).cria()
  module <module> line 234
    kmain(self)
NameError: name 'kmain' is not defined
'''},
{'date': 'Fri Aug 21 2020 09:27:19.822 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 397
    main(Jogo, STYLE)
  module <module> line 391
    Kwarwp(vitollino, medidas=medidas).cria()
  module <module> line 271
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 374
    self.o_indio = Indio(imagem, x=1, y=0, cena=cena, taba=self)
  module <module> line 74
    self.posicao = (x//lado,y//lado)
TypeError: unsupported operand type(s) for //: 'int' and 'NoneType'
'''},
{'date': 'Fri Aug 21 2020 09:29:08.198 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 397
    main(Jogo, STYLE)
  module <module> line 391
    Kwarwp(vitollino, medidas=medidas).cria()
  module <module> line 271
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 374
    self.o_indio = Indio(imagem, x=1, y=0, cena=cena, taba=self)
  module <module> line 75
    self.indio = Kwarwp.VITOLLINO.a(imagem, w=lado, h=lado, x=x, y=y, cena=cena)
AttributeError: 'NoneType' object has no attribute 'a'
'''},
{'date': 'Sat Aug 22 2020 09:25:30.675 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 400
    main(Jogo, STYLE)
  module <module> line 394
    Kwarwp(vitollino, medidas=medidas).cria()
  module <module> line 249
    fabrica = {
AttributeError: 'Kwarwp' object has no attribute 'indi0'
'''},
{'date': 'Wed Sep 09 2020 18:38:10.216 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: bases[i].$dict is undefined>
'''},
{'date': 'Wed Sep 09 2020 18:39:29.366 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: bases[i].$dict is undefined>
'''},
{'date': 'Wed Sep 09 2020 18:41:48.830 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: bases[i].$dict is undefined>
'''},
{'date': 'Wed Sep 09 2020 22:53:24.431 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
SyntaxError: invalid syntax

 module __ge739 line 2
  UTF8 -*-# Este arquivo é parte do programa Vi:
          ^
'''},
{'date': 'Wed Sep 09 2020 22:56:15.968 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
SyntaxError: invalid syntax

 module __ge739 line 2
  UTF8 -*-# Este arquivo é parte do programa Vi:
          ^
'''},
{'date': 'Wed Sep 09 2020 22:57:36.516 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
SyntaxError: invalid syntax

 module __ge1505 line 2
  TF8 -*-# Este arquivo é parte do programa Vit:
         ^
'''},
{'date': 'Wed Sep 09 2020 23:06:42.183 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: bases[i].$dict is undefined>
'''},
{'date': 'Wed Sep 09 2020 23:09:00.888 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module __ge739 line 2
  UTF8 -*-# Este arquivo é parte do programa Vi:
          ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Sep 09 2020 23:11:50.508 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module __ge739 line 2
  UTF8 -*-# Este arquivo é parte do programa Vi:
          ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Sep 09 2020 23:15:19.713 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
SyntaxError: invalid syntax

 module __ge739 line 2
  UTF8 -*-# Este arquivo é parte do programa Vi:
          ^
'''},
{'date': 'Mon Sep 14 2020 13:28:57.330 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  File __code/_core/supygirls_factory.py, line 135, in action
    <unknown>
  File __code/_core/supygirls_factory.py, line 310, in executa_acao
    <unknown>
  File __code/_core/supygirls_factory.py, line 268, in _first_response
    <unknown>
  File __code/_core/supygirls_factory.py, line 310, in <lambda>
    <unknown>
  File __code/_core/supygirls_factory.py, line 299, in _executa_acao
    <unknown>
  File __code/_core/supygirls_factory.py, line 153, in <module>
    main()
  File __code/_core/supygirls_factory.py, line 150, in main
    htmlrun.HTMLTestRunner().run(suite)
  File __code/kwarwp/htmlrunner.py, line 247, in run
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.suite, line 84, in __call__
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.suite, line 122, in run
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.case, line 736, in __call__
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.case, line 684, in run
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.case, line 600, in _feedErrorsToResult
    <unknown>
  File __code/kwarwp/htmlrunner.py, line 186, in addFailure
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.result, line 17, in addFailure
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.result, line 122, in addFailure
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.result, line 185, in _exc_info_to_string
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/traceback, line 509, in __init__
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/traceback, line 366, in extract
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/traceback, line 288, in line
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/linecache, line 16, in getline
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/linecache, line 47, in getlines
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/linecache, line 137, in updatecache
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/_io, line 517, in readlines
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/_io, line 1948, in __next__
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/_io, line 1969, in readline
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/_io, line 1647, in _get_decoder
    <unknown>
  File VFS.codecs.py, line 995, in getincrementaldecoder
    <unknown>
AttributeError: 'NoneType' object has no attribute 'incrementaldecoder'
'''},
{'date': 'Mon Sep 14 2020 13:30:15.530 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''  File <string>, line 20
    class Test_Kwarwp(TestCase)
                                ^
SyntaxError: invalid syntax 
'''},
{'date': 'Mon Sep 14 2020 13:31:39.953 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''  File <string>, line 20
    class Test_Kwarwp(TestCase)
                                ^
SyntaxError: invalid syntax 
'''},
{'date': 'Mon Sep 14 2020 13:33:36.967 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''  File <string>, line 24
    ABERTURA  "https://i.imgur.com/dZQ8liT.jpg"
               ^
SyntaxError: invalid syntax 
'''},
{'date': 'Mon Sep 14 2020 13:35:19.415 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  File __code/_core/supygirls_factory.py, line 20, in <module>
    """Brython front end client.
TypeError: Object passed as base class is not a class
'''},
{'date': 'Mon Sep 14 2020 13:37:56.920 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''  File <string>, line 21
    """ Teste do Jogo para ensino de programação.
         ^
SyntaxError: invalid syntax 
'''},
{'date': 'Mon Sep 14 2020 13:38:55.225 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''  File <string>, line 20
    class Test_Kwarwp(TestCase)
                                ^
SyntaxError: invalid syntax 
'''},
{'date': 'Mon Sep 14 2020 13:39:13.383 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  File __code/_core/supygirls_factory.py, line 135, in action
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  File __code/_core/supygirls_factory.py, line 310, in executa_acao
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  File __code/_core/supygirls_factory.py, line 268, in _first_response
    action()
  File __code/_core/supygirls_factory.py, line 310, in <lambda>
    
  File __code/_core/supygirls_factory.py, line 299, in _executa_acao
    exec(self.code, glob)  # dict(__name__="__main__"))
  File <string>, line 20, in <module>
TypeError: Object passed as base class is not a class
'''},
{'date': 'Mon Sep 14 2020 13:41:26.322 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 13:42:26.399 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 13:43:27.687 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 13:44:06.614 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': ''''''},
{'date': 'Mon Sep 14 2020 13:46:23.242 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  File __code/_core/supygirls_factory.py, line 135, in action
    <unknown>
  File __code/_core/supygirls_factory.py, line 310, in executa_acao
    <unknown>
  File __code/_core/supygirls_factory.py, line 268, in _first_response
    <unknown>
  File __code/_core/supygirls_factory.py, line 310, in <lambda>
    <unknown>
  File __code/_core/supygirls_factory.py, line 299, in _executa_acao
    <unknown>
Exception: <Javascript TypeError>: Cannot read property '$is_class' of undefined
'''},
{'date': 'Mon Sep 14 2020 13:46:46.528 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  File __code/_core/supygirls_factory.py, line 135, in action
    <unknown>
  File __code/_core/supygirls_factory.py, line 310, in executa_acao
    <unknown>
  File __code/_core/supygirls_factory.py, line 268, in _first_response
    <unknown>
  File __code/_core/supygirls_factory.py, line 310, in <lambda>
    <unknown>
  File __code/_core/supygirls_factory.py, line 299, in _executa_acao
    <unknown>
  File __code/_core/supygirls_factory.py, line 153, in <module>
    main()
  File __code/_core/supygirls_factory.py, line 150, in main
    htmlrun.HTMLTestRunner().run(suite)
  File __code/kwarwp/htmlrunner.py, line 247, in run
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.suite, line 84, in __call__
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.suite, line 122, in run
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.case, line 736, in __call__
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.case, line 684, in run
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.case, line 600, in _feedErrorsToResult
    <unknown>
  File __code/kwarwp/htmlrunner.py, line 186, in addFailure
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.result, line 17, in addFailure
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.result, line 122, in addFailure
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/unittest.result, line 185, in _exc_info_to_string
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/traceback, line 509, in __init__
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/traceback, line 366, in extract
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/traceback, line 288, in line
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/linecache, line 16, in getline
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/linecache, line 47, in getlines
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/linecache, line 137, in updatecache
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/_io, line 517, in readlines
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/_io, line 1948, in __next__
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/_io, line 1969, in readline
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/_io, line 1647, in _get_decoder
    <unknown>
  File https://cdn.jsdelivr.net/npm/brython@3.8.6/Lib/codecs, line 995, in getincrementaldecoder
    <unknown>
AttributeError: 'NoneType' object has no attribute 'incrementaldecoder'
'''},
{'date': 'Mon Sep 14 2020 13:49:11.547 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  File __code/_core/supygirls_factory.py, line 20, in <module>
    """Brython front end client.
TypeError: Object passed as base class is not a class
'''},
{'date': 'Mon Sep 14 2020 13:50:19.336 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  File __code/_core/supygirls_factory.py, line 20, in <module>
    """Brython front end client.
TypeError: Object passed as base class is not a class
'''},
{'date': 'Mon Sep 14 2020 13:50:36.841 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''  File <string>, line 20
    class Test_Kwarwp(TestCase)
                                ^
SyntaxError: invalid syntax 
'''},
{'date': 'Thu Sep 24 2020 12:35:36.360 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 607
    main(Jogo, STYLE)
  module <module> line 600
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 425
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 461
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i * lado, y=j * lado + lado, cena=cena)
  module <module> line 541
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
TypeError: __init__ missing 1 positional argument: 'taba'
'''},
{'date': 'Thu Sep 24 2020 12:39:15.663 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 601
    main(Jogo, STYLE)
  module <module> line 595
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 419
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 455
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 536
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
TypeError: __init__ missing 1 positional argument: 'taba'
'''},
{'date': 'Thu Sep 24 2020 12:41:56.425 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''False False <function _pos>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 601
    main(Jogo, STYLE)
  module <module> line 595
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 419
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 455
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 523
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa)
TypeError: __init__ missing 1 positional argument: 'taba'
'''},
{'date': 'Thu Sep 24 2020 13:19:34.649 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 46
    main()
TypeError: main() missing 2 positional arguments: vitollino,medidas
'''},
{'date': 'Mon Oct 05 2020 16:49:05.478 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 599
    main(Jogo, STYLE)
  module <module> line 593
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 416
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 452
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 508
    coisa.vazio.vai = lambda *_: self.o_indio.larga()
AttributeError: 'Nulo' object has no attribute 'vazio'
'''},
{'date': 'Mon Oct 05 2020 17:09:11.145 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 599
    main(Jogo, STYLE)
  module <module> line 593
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 416
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 452
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 506
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa, taba=self)
  module kwarwp.kwarwpart line 70
    self.acessa(ocupante)
  module kwarwp.kwarwpart line 121
    ocupante.ocupa(self)
  module kwarwp.kwarwpart line 227
    vaga.ocupou(self)
  module kwarwp.kwarwpart line 156
    self.vazio.ocupa(ocupante, pos)
TypeError: ocupa() takes 2 positional arguments but more were given
'''},
{'date': 'Mon Oct 05 2020 17:33:47.693 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 602
    main(Jogo, STYLE)
  module <module> line 596
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 415
    Kwarwp.KW.fala(Vazio.LADO)
AttributeError: 'NoneType' object has no attribute 'fala'
'''},
{'date': 'Mon Oct 05 2020 17:34:32.191 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 602
    main(Jogo, STYLE)
  module <module> line 596
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 419
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 428
    Kwarwp.KW.fala(Vazio.LADO)
AttributeError: 'NoneType' object has no attribute 'fala'
'''},
{'date': 'Mon Oct 05 2020 17:35:13.768 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 602
    main(Jogo, STYLE)
  module <module> line 596
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 419
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 428
    Kwarwp.KW.fala(Vazio.LADO)
  module <module> line 465
    self.ceu.elt.html = texto
AttributeError: 'Kwarwp' object has no attribute 'ceu'
'''},
{'date': 'Mon Oct 05 2020 18:12:55.719 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 608
    main(Jogo, STYLE)
  module <module> line 602
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 423
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 433
    fabrica = {
AttributeError: 'Kwarwp' object has no attribute 'apedra'
'''},
{'date': 'Mon Oct 05 2020 18:59:37.18 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''False False <function _pos>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 656
    main(Jogo, STYLE)
  module <module> line 650
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 457
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 494
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 550
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa, taba=self)
  module kwarwp.kwarwpart line 70
    self.acessa(ocupante)
  module kwarwp.kwarwpart line 121
    ocupante.ocupa(self)
  module kwarwp.kwarwpart line 302
    self.empurrante.fala(self.empurrante.indio.pos)
AttributeError: 'Nulo' object has no attribute 'fala'
'''},
{'date': 'Mon Oct 05 2020 19:13:05.273 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''False False <function _pos>
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 646
    main(Jogo, STYLE)
  module <module> line 640
    return Kwarwp(vitollino=vitollino_proxy, medidas=medidas, mapa=mapa, indios=indios)
  module <module> line 447
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 484
    self.taba = {(i, j): fabrica[imagem].objeto(fabrica[imagem].imagem, x=i*lado, y=j*lado+lado, cena=cena)
  module <module> line 540
    vaga = Vazio("", x=x, y=y, cena=cena, ocupante=coisa, taba=self)
  module kwarwp.kwarwpart line 70
    self.acessa(ocupante)
  module kwarwp.kwarwpart line 121
    ocupante.ocupa(self)
  module kwarwp.kwarwpart line 302
    self.empurrante.fala(self.empurrante.indio.pos)# if self.empurrante is not NULO else None
AttributeError: 'Nulo' object has no attribute 'fala'
'''},